menuconfig/configs/opensips_residential_def.m4

```diff
@@ -1,6 +1,6 @@
 divert(-1)
-define(`ENABLE_TCP', `no') # OpenSIPS will listen on TCP for SIP requests
-define(`ENABLE_TLS', `no') # OpenSIPS will listen on TLS for SIP requests
+define(`ENABLE_TCP', `yes') # OpenSIPS will listen on TCP for SIP requests
+define(`ENABLE_TLS', `yes') # OpenSIPS will listen on TLS for SIP requests
```

Comment:
This is not related to postgresql and I don't think we should enable TLS/TCP by default in the residential script anyway.